MzmlParser/MzmlReader.cs

```diff
@@ -110,7 +110,7 @@ public Run LoadMzml(string path, AnalysisSettings analysisSettings)
         private void ReadMzml(string path, Run run, bool irt)
         {
 
-            using (XmlReader reader = XmlReader.Create(path))
+            using (XmlReader reader = "-".Equals(path) ? XmlReader.Create(Console.In): XmlReader.Create(path))
```

Comment:
Can we put in an explantory comment here?

Reply:
No string culture on this one but there is on the other one?

Reply:
Added explanatory comments and correct StringComparisons.